src/store/components/AdminPickupPage/index.tsx

```diff
@@ -165,6 +166,26 @@ const AdminPickupPage: React.FC<AdminPickupPageProps> = (props) => {
                   <StoreButton type="secondary" size="medium" text="Cancel" disabled={submitting} onClick={() => setCreateMode(false)} />
                   <StoreButton text="Save" size="medium" disabled={submitting} onClick={() => handleSubmit()} />
                 </div>
+                {!creatingPickup && (
+                  <div className="admin-pickup-page-buttons">
+                    <StoreButton
+                      type="danger"
+                      size="medium"
+                      text="Delete"
+                      disabled={submitting}
+                      onClick={() => {
+                        deletePickupEvent(pickupEvent?.uuid)
+                          .then(() => {
+                            notify('Success!', `Deleted ${pickupEvent?.title}.`);
+                            history.push('/store/admin');
+                          })
+                          .catch((reason) => {
+                            notify('API Error', reason.message || reason);
+                          });
```

Comment:
Is it common in the repo to inline multiline functions as a prop? I find these can make the JSX markup hard to read sometimes.

Reply:
Good point; I pulled this out into a separate `handleDelete` function.